graphql-dgs-client/src/main/kotlin/com/netflix/graphql/dgs/client/GraphQLClient.kt

```diff
@@ -92,3 +190,28 @@ fun interface MonoRequestExecutor {
  */
 class GraphQLClientException(statusCode: Int, url: String, response: String, request: String) :
     RuntimeException("GraphQL server $url responded with status code $statusCode: '$response'. The request sent to the server was \n$request")
+
+internal object GraphQLClients {
+    internal val objectMapper: ObjectMapper = try {
```

Comment:
We already bring Spring in the client and it has the ClassUtils class with `public static boolean isPresent(String className, @Nullable ClassLoader classLoader) `.